packages/hooks/src/types.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal dependencies
+ */
+import type { Hooks as internalHooks } from './createHooks';
```

Comment:
Why not simply

```suggestion
/**
export type { Hooks } from './createHooks';
```

Then remove the export below.